govcd/api_vcd_versions.go

```diff
@@ -317,12 +317,14 @@ func (client *Client) GetVcdFullVersion() (VcdVersion, error) {
 	if err != nil {
 		return VcdVersion{}, err
 	}
+	// VCFA 9.1+ adds extra "customer patch" version that does not follow SemVer
+	version = strings.ReplaceAll(version, "-", ".")
 
 	vcdVersion.Version, err = semver.NewVersion(version)
 	if err != nil {
 		return VcdVersion{}, err
 	}
-	if len(vcdVersion.Version.Segments()) < 4 {
+	if len(vcdVersion.Version.Segments()) < 3 {
```

Comment:
Not needed, but we should probably be less restrictive and consider only major.minor.patch as relevant.